ang/test/civicase/Dashboard.spec.js

```diff
@@ -71,7 +71,7 @@
 
         it('filters the cases and activties where the user is involved', function () {
           expect($scope.activityFilters.case_filter).toEqual(jasmine.objectContaining({
-            contact_id: [ CRM.config.user_contact_id ]
+            contact_involved: {'IN': [CRM.config.user_contact_id]}
```

Comment:
Why do we need to introduce `IN` clause. Is there a reason for it?

Reply:
This is because the implementation was buggy before. As the `Cases I am involved in` filter never returned any results. Because when passing an array as a param, `IN` key needs to be used. 

In `Search.js` specifying the `IN` operator was not required, because it is taken care of in https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase/CaseListTable.js#L298